menu_after_form/quiz/media_queries_quizHTML/phone_screen.css

```diff
@@ -0,0 +1,133 @@
+@media (min-width: 320px) and (max-width: 540px) {
```

Comment:
Same as above, duplicated css in separated files. I'm not sure, if you are loading only one file if you detect actual screen size. If you are loading only one file, it's ok, if you are loading multiple files, it should be stored in one file